docker/unified-builder

```diff
@@ -150,3 +154,15 @@ FROM debian:bullseye-slim AS chronicle-tp
 ARG TARGETARCH
 WORKDIR /
 COPY .artifacts/artifacts/${TARGETARCH}/chronicle_sawtooth_tp /usr/local/bin
+
```

Comment:
Since opa-tp is destined to become a separate repo, it is best if this were a full separate builder

Reply:
Ok, I was going to split that out in when needed as it still ends up in the makefile, but can do this now